src/FEValues/common_values.jl

```diff
@@ -53,11 +55,23 @@ Return the number of quadrature points in `fv`s quadrature for the current
 """
 getnquadpoints(fe::FaceValues) = getnquadpoints(fe.qr, fe.current_face[])
 
+"""
+    getnquadpoints(iv::InterfaceValues)
+
+Return the number of quadrature points in `iv`s element A's [`FaceValues`](@ref)' quadrature for the current
+(most recently [`reinit!`](@ref)ed) interface.
+"""
+getnquadpoints(iv::InterfaceValues) = getnquadpoints(iv.face_values_a.qr, iv.face_values_a.current_face[])
+
 """
     getdetJdV(fe_v::AbstractValues, q_point::Int)
+    getdetJdV(iv::InterfaceValues, q_point::Int; use_element_a::Bool)
```

Comment:
Would the weight not be the same anyway? Why is it useful to specify which side in this case?

Reply:
I think detj can be different

Reply:
If you have
```
+-----+---+
|      |    |
+-----+---+
```
then detJ differs between the elements already.

Reply:
Does that imply that incompatible/discontinuous geometric meshes are allowed (in addition to discontinuous fields)? If so, how does that work? I thought the quadrature points have to be in the same physical location to calculate the jump (at least in the initial configuration if considering finite strains).

Reply:
If I understand correctly, I think it can work if we disable the nodes check here:

https://github.com/Ferrite-FEM/Ferrite.jl/blob/d82f177fe86d0998f5d32fb9c300222d3cbe4e91/src/Grid/grid.jl#L1658

the quadrature points don't have the same physical location (but are the same type?), like 
```
+-----+    +---+
|     |    |   |
+-----+    +---+
```
so we have the same quadrature rule for the faces but they are not at the same location.

Reply:
Ah, got it. It currently doesn't support this case (we assume each face may have one neighbor only or none).

Reply:
Correction: Fredrik is right, we use the weight of the face mapping and not the determinant of the cell mapping.

Reply:
It should be possible to handle the case
```
+-----++---+
|     ||   |
+-----+|   |
|     ||   |
+-----++---+
```
by introducing sub-faces or by other mechanisms. But we leave this handling to subsequent PRs. We need more time from our side (Maxi+me) for an efficient forest mesh implementation.